keyboards/j80/info.json

```
@@ -0,0 +1,202 @@
+{
+  "keyboard_name": "J80",
+  "url": "",
+  "maintainer": "JER",
+  "width": 18.25,
+  "height": 6.25,
+  "layouts": {
+    "LAYOUT_tkl_ansi": {
+      "layout": [
+        {"x":0, "y":0},
+        {"x":2, "y":0},
+        {"x":3, "y":0},
+        {"x":4, "y":0},
+        {"x":5, "y":0},
+        {"x":6.5, "y":0},
+        {"x":7.5, "y":0},
+        {"x":8.5, "y":0},
+        {"x":9.5, "y":0},
+        {"x":11, "y":0},
+        {"x":12, "y":0},
+        {"x":13, "y":0},
+        {"x":14, "y":0},
+        {"x":15.25, "y":0},
+        {"x":16.25, "y":0},
+        {"x":17.25, "y":0},
+
+        {"x":0, "y":1.25},
+        {"x":1, "y":1.25},
+        {"x":2, "y":1.25},
+        {"x":3, "y":1.25},
+        {"x":4, "y":1.25},
+        {"x":5, "y":1.25},
+        {"x":6, "y":1.25},
+        {"x":7, "y":1.25},
+        {"x":8, "y":1.25},
+        {"x":9, "y":1.25},
+        {"x":10, "y":1.25},
+        {"x":11, "y":1.25},
+        {"x":12, "y":1.25},
+        {"x":13, "y":1.25, "w":2},
+        {"x":15.25, "y":1.25},
+        {"x":16.25, "y":1.25},
+        {"x":17.25, "y":1.25},
+
+        {"x":0, "y":2.25, "w":1.5},
+        {"x":1.5, "y":2.25},
+        {"x":2.5, "y":2.25},
+        {"x":3.5, "y":2.25},
+        {"x":4.5, "y":2.25},
+        {"x":5.5, "y":2.25},
+        {"x":6.5, "y":2.25},
+        {"x":7.5, "y":2.25},
+        {"x":8.5, "y":2.25},
+        {"x":9.5, "y":2.25},
+        {"x":10.5, "y":2.25},
+        {"x":11.5, "y":2.25},
+        {"x":12.5, "y":2.25},
+        {"x":13.5, "y":2.25, "w":1.5},
+        {"x":15.25, "y":2.25},
+        {"x":16.25, "y":2.25},
+        {"x":17.25, "y":2.25},
+
+        {"x":0, "y":3.25, "w":1.75},
+        {"x":1.75, "y":3.25},
+        {"x":2.75, "y":3.25},
+        {"x":3.75, "y":3.25},
+        {"x":4.75, "y":3.25},
+        {"x":5.75, "y":3.25},
+        {"x":6.75, "y":3.25},
+        {"x":7.75, "y":3.25},
+        {"x":8.75, "y":3.25},
+        {"x":9.75, "y":3.25},
+        {"x":10.75, "y":3.25},
+        {"x":11.75, "y":3.25},
+        {"x":12.75, "y":3.25, "w":2.25},
+
+        {"x":0, "y":4.25, "w":2.25},
+        {"x":2.25, "y":4.25},
+        {"x":3.25, "y":4.25},
+        {"x":4.25, "y":4.25},
+        {"x":5.25, "y":4.25},
+        {"x":6.25, "y":4.25},
+        {"x":7.25, "y":4.25},
+        {"x":8.25, "y":4.25},
+        {"x":9.25, "y":4.25},
+        {"x":10.25, "y":4.25},
+        {"x":11.25, "y":4.25},
+        {"x":12.25, "y":4.25, "w":2.75},
```

Changing this to Split Right Shift.

```suggestion
        {"x":12.25, "y":4.25, "w":1.75},
        {"x":14, "y":4.25},
```